src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+export function list(settings, logger) {
+  const files = fs.readdirSync(settings.pluginDir);
+  files
```

Comment:
I apologize for not catching this in the first go around, but I didn't notice it until you chained these together.  If you're chaining anyway, you might as well ditch the local variable entirely:

```js
fs.readdirSync(settings.pluginDir)
.filter(...)
.forEach(...)
```